ci-operator/config/rh-ecosystem-edge/ci-artifacts/rh-ecosystem-edge-ci-artifacts-master.yaml

```diff
@@ -0,0 +1,28 @@
+base_images:
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
```

Comment:
Or 1.17 ?

Reply:
1.16 currently just like PSAP ci-artifacts. Changes will come after this merge